packages/image_picker/example/pubspec.yaml

```diff
@@ -9,10 +9,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  image_picker: ^0.8.6
+  image_picker: ^1.0.1
   image_picker_tizen:
     path: ../
-  video_player: ">=2.1.4 <2.7.0"
+  video_player: ^2.4.2
+  video_player_tizen: ^2.4.7
 
```

Comment:
You need to add mime package dependency.
https://github.com/flutter/packages/blob/main/packages/image_picker/image_picker/example/pubspec.yaml#L21C15-L21C15

Reply:
Added mime package dependency. thank you.